accounting/accounting_cli.py

```diff
@@ -124,6 +127,18 @@ def main():
         "end_time", help="end time", metavar="END TIME",
     )
 
+    subparser_add_bank = subparsers.add_parser("add-bank", help="add a new bank")
+    subparser_add_bank.set_defaults(func="add_bank")
+    subparser_add_bank.add_argument(
+        "--bank", help="bank name", metavar="BANK",
```

Comment:
Same comment as before. Would it be more consistent to make this a mandatory argument? If you want to keep it this way, maybe you can change the way you specify `delete-bank` instead... As far as you make this consistent, this works for me.

Reply:
Agreed. I think both `bank` and `shares` should be mandatory arguments; `parent-bank` will be an optional argument. I'll make this change now.